src/logging.c

```diff
@@ -135,12 +146,64 @@ void rcutils_logging_console_output_handler(
   }
 
   if (!location) {
-    fprintf(stream, "[%s] [%s]: %s\n", severity_string, name, message_buffer);
-  } else {
-    fprintf(
-      stream, "[%s] [%s]: %s (%s() at %s:%zu)\n", severity_string, name, message_buffer,
-      location->function_name, location->file_name, location->line_number);
+    // TODO before merge
+    //location = empty_location;
+  }
+
+  // Process the format string looking for known tokens
+  char token_start_delimiter = '{';
+  char token_end_delimiter = '}';
+  char output_buffer[1024];
+  memset(output_buffer, '\0', sizeof(output_buffer));
+  const char * str = g_rcutils_logging_output_format_string;
+  size_t size = strlen(g_rcutils_logging_output_format_string);
+
+  for (size_t i = 0; i < size; ++i) {
+    if (str[i] == token_start_delimiter) {
```

Comment:
I would invert the check since allows you to indent the whole following block:

```
if (str[i] != token_start_delimiter) {
  strncat(output_buffer, str + i, 1);
  continue;
}

char token[20];
...
```